resources/metadata/abapEnvironmentRunATCCheck.yaml

```diff
@@ -0,0 +1,101 @@
+metadata:
+  name: abapEnvironmentRunATCCheck
+  description: Runs an ATC Check
+  longDescription: |
+    Run ATC Check
+spec:
+  inputs:
+    secrets:
+      - name: cfCredentialsId
```

Comment:
I would keep this consistent with abapEnvironmentPullGitRepo. As the credentials could either be for cloudFoundry or the ABAP instance itself, I would not call it "cfCredentialsId".
I would choose the generic name "credentialsId"

Reply:
Good suggestion. 👍